### PR TITLE
[MIG] website_sale_secondary_unit : Migration to 14.0

### PR DESCRIPTION
**Pull request by kimirondon**

### CODE REVIEW

**Comment by kimirondon** [comment]
depends on this PR https://github.com/OCA/sale-workflow/pull/1428

**Comment by sergio-teruel** [comment]
@kimirondon please can yo add this code from v13.0
* https://github.com/OCA/e-commerce/pull/558

**Comment by kimirondon** [comment]
@sergio-teruel done!
the checks are failing because of the dependency with module sale_order_secondary_unit, waiting for this PR https://github.com/OCA/sale-workflow/pull/1671

**Comment by pedrobaeza** [comment]
Please include #573

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 # See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+sale-workflow https://github.com/archeti-org/sale-workflow 14.0-mig-sale_order_secondary_unit
```
Comment (CarlosRoca13) [inline oca_dependencies.txt]:
```suggestion
sale-workflow
```

**Review by CarlosRoca13 — COMMENTED** [review]
I think that this is blocking

File: oca_dependencies.txt (hunk 1 above)
Comment (kimirondon) [reply oca_dependencies.txt]: Hello! 
I updated the oca_dependencies.txt file as you recommend me but still have this issue:
ERROR: Could not find a version that satisfies the requirement odoo14-addon-sale_order_secondary_unit (from versions: none)

**Comment by pedrobaeza** [comment]
Please include #641 and make Travis green.

**Comment by kimirondon** [comment]
@pedrobaeza 
hello! it's done I added the commit from #641, the checks fails due to a dependency on sale_order_secondary_unit which hasn't been migrated yet, PR https://github.com/OCA/sale-workflow/pull/1671

**Comment by CarlosRoca13** [comment]
Please add this PR https://github.com/OCA/e-commerce/pull/653

**Comment by kimirondon** [comment]
hello @CarlosRoca13  done!

**Review by chienandalu — COMMENTED** [review]
@kimirondon rebase with current `14.0` head to get new CIs

**Comment by francesco-ooops** [comment]
@kimirondon can you rebase?

**Comment by kimirondon** [comment]
hello @francesco-ooops!  it's done.

**Comment by francesco-ooops** [comment]
> hello @francesco-ooops!  it's done.
> 

Thanks! Can you try to fix runboat for functional review?

**Comment by chienandalu** [comment]
Please include this changes: https://github.com/OCA/e-commerce/pull/712

**Comment by kimirondon** [comment]
hello @chienandalu ! done I added your commit.

**Review by chienandalu — COMMENTED** [review]
Tests are failing

**Comment by kimirondon** [comment]
hello @chienandalu ! I updated and rebase my branch. The tests are failing due a missing field (active) in the model "product.secondary.unit", this field exist in v13 and also in v15 but not in v14. Should I modify my controller or should I wait until this is fixed in v14 ?

https://github.com/OCA/product-attribute/blob/13.0/product_secondary_unit/models/product_second_unit.py#L27
https://github.com/OCA/product-attribute/blob/15.0/product_secondary_unit/models/product_second_unit.py#L45C42-L45C42

**Comment by chienandalu** [comment]
Well, it's been a long time since this was opened and several things could change in the meantime :sweat_smile: 

I'd check for missing fw-ports of changes to v14 like the one you spotted in the dependencies of this module first. You can use the great [oca-port](https://github.com/OCA/oca-port) tool for it.

**Comment by kimirondon** [comment]
hello @chienandalu ! thanks a lot for the help, I used oca-port as you recommended, it work really nice.
this is the corresponding PR 
https://github.com/OCA/product-attribute/pull/1388

**Pull request closed without merging**